src/components/BrandConfigProvider/styles/createStyles.ts

```diff
@@ -52,6 +54,8 @@ type TextSuffixes = 'Text' | 'Label';
 
 type ImageSuffixes = 'Image';
 
+type AnySuffix = 'Any';
```

Comment:
Cheat code for allowing a child components styles within a parents styles: https://github.com/lifeomic/react-native-sdk/pull/171/files#diff-83f4bf1929726afb337b4ad9f7c588a6a8165205ed8f177be6490e058638dfacR85

Ideally we'd have some magic to simply infer the type from the set property but I'm not sure that is doable (or worth doing).

Reply:
Interesting solution.  However, is it really needed?  I have passed styles to child components before without issue.

Reply:
If you want to embed a child component's styles within a parent's default styles, you will need to redefine each style type (e.g., childView, childLabel, etc.) and pass them accordingly. This is because our style types only support basic component types, while the child components contain nested styles. Unless there's another approach I am not considering, this `Any` hack allows us to both define the styles in the parent's default styles and pass them in directly to the child component.

Reply:
Note that I'm not I'm not actually sure if it is needed or not.  Approving incase I'm wrong.  👍

Reply:
I may revisit this to see if I can come up with a better idea in a future PR. Merging as-is for now.